script/example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import yaml
+import json
+from argparse import ArgumentParser
+
+def action_json(args):
+
+    with open('scripts/example_data.json') as json_file:
+        print(json.dumps(json.loads(json_file.read()), indent=2))
+
+def action_yaml(args):
+    with open('scripts/example_data.yaml') as yaml_file:
```

Comment:
Similar to the previous comment, use a context manager for handling YAML files to ensure proper resource management.
```suggestion
with open('scripts/example_data.yaml', 'r') as yaml_file:
```